.github/workflows/packaging-pipeline.yml

```diff
@@ -0,0 +1,57 @@
+name: OpenCue Packaging Pipeline
+
+# Trigger this pipeline on new commits to master.
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  # TODO(https://github.com/AcademySoftwareFoundation/OpenCue/issues/715) Add the other
+  #   packaging steps here. Things like building Docker images and extracting artifacts
+  #   will need to be generalized, but we need to test a few things in the master branch
+  #   first to make sure things are working as we expect.
+  build_cuebot:
+    name: Build Cuebot
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set build ID
+        run: |
+          ci/generate_version_number.sh > VERSION
+          echo "::set-env name=BUILD_ID::$(cat ./VERSION)"
+      - uses: docker/build-push-action@v1
+        name: Build Docker image
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASS }}
+          dockerfile: cuebot/Dockerfile
+          repository: opencuebuild/cuebot
+          tags: ${{ env.BUILD_ID }}
+      - name: Extract Artifacts
+        run: |
+          set -e
+          image_name="opencuebuild/cuebot:${BUILD_ID}"
+          container_id=$(docker create ${image_name})
+          artifacts="/opt/opencue/cuebot-${BUILD_ID}-all.jar /opt/opencue/opencue-cuebot-${BUILD_ID}-1.noarch.rpm"
+          mkdir -p "${GITHUB_WORKSPACE}/artifacts/"
+          for artifact in $artifacts; do
+            docker cp ${container_id}:${artifact} "${GITHUB_WORKSPACE}/artifacts/"
+          done
+          docker rm $container_id
+      - uses: actions/upload-artifact@v2
+        name: Upload JAR
+        with:
+          name: cuebot-jar
+          path: ${{ github.workspace }}/artifacts/cuebot-*.jar
+      - uses: actions/upload-artifact@v2
+        name: Upload RPM
+        with:
+          name: cuebot-rpm
+          path: ${{ github.workspace }}/artifacts/opencue-cuebot-*.rpm
+      # This step is temporary. This info will help us track this pipeline's artifacts as we
```

Comment:
I'd like a more clear description of when this step will be removed.

Reply:
Added more info -- coming out in the next PR, basically -- and made it a TODO.